### ISSUE TITLE
TS12-P3 - ML Challenge Page / Challenge Launch version ---- Copy Update

### ISSUE THREAD

**Issue by kev-zunshiwang** [chanzuckerberg/cryoet-data-portal#1194]
Janeece's update as of 9/30:

- The pre-challenge launch version was finished but is on hold due to Kaggle. ([Specs here](https://www.figma.com/design/aL3EkC3uM17oFDmTC5h4S8/Janeece%2F-Kevin---ML-Challenge-Page?node-id=762-3331&t=0p96IprMOEg2Q8n0-1))
- Launch version is waiting on final copy, but we added a few placeholder sections back in ([WIP design here](https://www.figma.com/design/aL3EkC3uM17oFDmTC5h4S8/Janeece%2F-Kevin---ML-Challenge-Page?node-id=706-3101&t=0p96IprMOEg2Q8n0-1) in the original location)
- Will need your help when final copy is ready to update and prep hand-off
- when it is handed off, [move to its own page here](https://www.figma.com/design/aL3EkC3uM17oFDmTC5h4S8/Janeece%2F-Kevin---ML-Challenge-Page?node-id=762-3332&t=0p96IprMOEg2Q8n0-1) (so eng is clear on what version they are implementing)

**Comment by Janeece** [comment]
@kev-zunshiwang is this complete? can you close it if so?
